packages/Automatic Differentiation

```diff
@@ -0,0 +1,45 @@
+---
+layout: "package"
+permalink: "https://github.com/StevenWaldrip/Automatic-Differentiation"
+description: >-
+  Automatic-Differentiation for Octave.
+icon: "https://raw.githubusercontent.com/gnu-octave/pkg-example/main/doc/icon.png"
```

Comment:
If you have an icon that could represent your package, you could point to it here.
This is the icon that is shown for the package in the index.
Currently, that is this icon:
https://raw.githubusercontent.com/gnu-octave/pkg-example/main/doc/icon.png